RecoTracker/LSTCore/standalone/README.md

```diff
@@ -106,186 +153,57 @@ Comparing two different runs
         -t "mywork" \
         --compare
 
-## CMSSW Integration
-This is the a complete set of instruction on how the TrackLooper code
-can be linked as an external tool in CMSSW:
+# How to set up CMSSW with LST
 
-### Build TrackLooper
-```bash
-git clone git@github.com:SegmentLinking/TrackLooper.git
-cd TrackLooper/
-# Source one of the commands below, depending on the site
-source setup.sh # if on UCSD or Cornell
-source setup_hpg.sh # if on Florida
-sdl_make_tracklooper -mc
-cd ..
-```
+## Setting up the area
```

Comment:
Do we even need this section? Isn't that already covered above ("Setting up LST" section)?

...I know, probably a question to myself back in the day, but maybe we can improve now...